src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -56,3 +56,23 @@ CREATE TABLE "file" (
   uploaded_by_user_id UUID REFERENCES "user" (user_id),
   created_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
 );
+
+CREATE TABLE "iep" (
+  iep_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+  student_id UUID REFERENCES "student" (student_id),
+  case_manager_id UUID REFERENCES "user" (user_id),
+  start_date DATE,
+  end_date DATE
+);
+
+CREATE TABLE "goal" (
+  goal_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(), -- TODO: add index to allow reordering
+  iep_id UUID REFERENCES "iep" (iep_id),
+  description TEXT
+);
+
+CREATE TABLE "subgoal" (
+  subgoal_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(), -- TODO: add index to allow reordering
+  goal_id UUID REFERENCES "goal" (goal_id),
+  description TEXT -- TODO: add more fields
+);
```

Comment:
would recommend adding `created_at` to all tables (see above for an example) and adding `NOT NULL` constraints where appropriate

Reply:
Good suggestions. Added `created_at` and `NOT NULL` constraints. At some point, we should probably implement the full history for our tables.